docs/_snippets/common-links.md

```diff
@@ -27,7 +27,7 @@
 [transaction result codes]: https://xrpl.org/docs/references/protocol/transactions/transaction-results
 [universal error types]: https://xrpl.org/docs/references/http-websocket-apis/api-conventions/error-formatting#universal-errors
 <!-- TODO: replace with link to roadmap page when that's ready -->
-[Lending Protocol amendment]: https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0066d-lending-protocol
+[Lending Protocol amendment]: https://github.com/Tapanito/XRPL-Standards/tree/xls-66-lending-protocol/XLS-0066d-lending-protocol
```

Comment:
Typically at the Devnet stage, the spec is merged and the docs link to the XRPL-Standards master branch. Is there a reason why this is different?